perun-wui-core/src/main/java/cz/metacentrum/perun/wui/model/beans/Attribute.java

```diff
@@ -148,6 +148,27 @@ public native final String getGuiUniqueId() /*-{
 		return JsonUtils.parseJsonToMap(getValueAsJso());
 	}
 
+	/**
+	 * Returns true, if attribute value is empty
+	 *
+	 * @return true if empty, false otherwise
+	 */
+	public final boolean isEmpty() {
```

Comment:
Please add support for ``java.lang.LargeString`` and ``java.util.LargeArrayList`` attribute value types into the whole Attribute and AttributeDefinition classes. They should work just like string and array, they just can contain more than 4000 chars (bytes) on Oracle DB.

Reply:
Added.